POS/Integration/08_ManageWarehouse.md

```diff
@@ -1,19 +1,19 @@
 [!!Configure the warehouse for POS](./01_ConfigureWarehouse.md)
 
 [comment]: <> (add link to Warehouse if available)
-
+<!---Marina, kannst Du hier gucken wegen Terminologie,ob es das so überhaupt noch gibt und ob man links setzen kann?-->
 # Manage the POS warehouse
 
-Manage the warehouse settings to assign one or more specific warehouses to the POS from which the stock is taken and booked. The warehouse specifies where exactly the stock is booked out when a product is sold.
+Manage the warehouse settings to assign one or more specific warehouses to the POS from which the stock is taken from and posted. The warehouse specifies where exactly the stock is taken from when a product is sold.
 
 ## Check the stock allocation
 
-Check if the stock allocation is correctly configured so that the store stock is correctly assigned to the warehouse. The stock source specifies how the stock available in the POS system is calculated. In most cases, the stock source and the warehouse are identical, but they can also differ.   
+Check if the stock allocation is correctly configured so that the store stock level is correctly assigned to the warehouse. The stock source specifies how the stock available in the POS system is calculated. In most cases, the stock source and the warehouse are identical, but they can also differ.   
```

Comment:
Hier tendiere ich eher to "stock value" statt "stock level", denn es ist  berechnet in the stock allocation matrix.

Reply:
done